tests/test_bank/test_wavebank.py

```diff
@@ -339,6 +339,12 @@ def test_empty_bank_raises(self, tmpdir):
         bank.put_waveforms(obspy.read())
         assert len(bank.read_index()) == 3
 
+    def test_update_index_returns_self(self, default_bank):
+        """ ensure update index returns the instance for chaining. """
+        out = default_bank.update_index()
+        assert out is not None
```

Comment:
Does the next line make this line unnecessary?

Reply:
Ya, it could easily be removed.